Gemfile.jruby-3.1.lock.release

```diff
@@ -964,7 +964,7 @@ DEPENDENCIES
   logstash-integration-rabbitmq
   logstash-integration-snmp
   logstash-output-csv
-  logstash-output-elasticsearch (>= 11.14.0)
+  logstash-output-elasticsearch
```

Comment:
This change seems to me accidental, `logstash-output-elasticsearch` is removed from `Gemfile.template` by https://github.com/elastic/logstash/pull/16984
@jsvd safe to move forward or need to put back in Gemfile.template?

Reply:
well spotted, please add it back.